src/styles/nav.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+  margin: 0;
+}
+
+.link__adress {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+  margin-right: 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+  cursor: pointer;
+
+  &,
+  &:hover {
+    transition: 0.3s;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+
+  &:after {
+    position: absolute;
+    content: "";
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    bottom: 0;
+  }
+
+}
+
+.link__adress:hover {
+  color: #00acdc;
+}
+
+.nav__link:last-of-type .link__adress {
+  margin-right: 0;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    height: 40px;
```

Comment:
use mixin here do declare heigth an width - because now your style is good only because you have picture 40*40px but if it will change you can have not clear result